project_event/models/project_task.py

```diff
@@ -227,6 +227,14 @@ class Task(models.Model):
         string='Actual Total Time',
         compute='_compute_actual_total_time',
     )
+    name_code_pair = fields.Char(
```

Comment:
We already have  a compute field : complete_name. This field concatenates the code/name

Reply:
@rimbendhaou Changed